sig/app/models/collaboration.rbs

```diff
@@ -0,0 +1,2 @@
+class Collaboration
+end
```

Comment:
Just a little concern, do we need these empty class?

Reply:
Assume, From a method of User class it's returning Array[Project]. 
For the same purpose, We can keep this class , so that we can use it later for return  type of other methods.
Maybe in the models RBS file this class is not referred anywhere. but in other part of the source code , it can be referred. So i think, we need to keep that

Reply:
@tanmoysrt Add empty classes when needed, remove empty classes which aren't referenced yet